contrib/k8s/dex-overlord.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dex
+type: Opaque
+data:
+  key-secrets: ZUhoNGVIaDRlSGg0ZUhoNGVIaDRlSGg0ZUhoNGVIaDRlSGg0ZUhoNGVIZz0= # 32 x's base64 encoded twice.
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: dex
+    role: overlord
+  name: dex-overlord
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: dex
+        role: overlord
+    spec:
+      containers:
+          - image: quay.io/coreos/dex
+            name: dex-overlord
+            env:
+              - name: DEX_OVERLORD_DB_URL
+                value: postgres://postgres@dex-postgres:5432/postgres?sslmode=disable
+              - name: DEX_OVERLORD_ADMIN_LISTEN
+                value: http://0.0.0.0:5557
+              - name: DEX_SECRET
```

Comment:
If you rename this to `DEX_OVERLORD_KEY_SECRETS` you can avoid passing the flag below.